roles/bind9/molecule/default/verify.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: check bind9 package status
+    package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
+    loop:
+      - bind9
+
+  - name: test for bind9 packages
+    assert:
+      that:
+        - not pkg_status.changed
+
+  - name: check status of bind9
+    service:
+      name: bind9
+      state: started
+    check_mode: true
```

Comment:
Do you need an assert here?

Reply:
I was blindly bringing in the tests from the >= 2.22